scanner/src/main/java/edu/ucr/cs/riple/scanner/ErrorProneCLIFlagsConfig.java

```diff
@@ -46,34 +48,26 @@ public class ErrorProneCLIFlagsConfig implements Config {
   static final String EP_FL_NAMESPACE = "Scanner";
   static final String FL_CONFIG_PATH = EP_FL_NAMESPACE + ":ConfigPath";
 
-  public ErrorProneCLIFlagsConfig() {
-    this.methodTrackerIsActive = false;
-    this.fieldTrackerIsActive = false;
-    this.callTrackerIsActive = false;
-    this.classTrackerIsActive = false;
-    this.outputDirectory = null;
-    this.serializer = new Serializer(this);
-  }
-
   public ErrorProneCLIFlagsConfig(ErrorProneFlags flags) {
     String configFilePath = flags.get(FL_CONFIG_PATH).orElse(null);
-    Document document = null;
-    if (configFilePath != null) {
-      try {
-        DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-        DocumentBuilder builder = factory.newDocumentBuilder();
-        document = builder.parse(Files.newInputStream(Paths.get(configFilePath)));
-        document.normalize();
-      } catch (IOException | SAXException | ParserConfigurationException e) {
-        throw new RuntimeException("Error in reading/parsing config at path: " + configFilePath, e);
-      }
+    Preconditions.checkNotNull(
+        configFilePath,
+        "Error in Scanner Checker configuration, should be set with via error prone flag: (-XepOpt:Scanner:ConfigPath)");
```

Comment:
Do we have a test for this? We probably want a test case that shows that an error is thrown if `FL_CONFIG_PATH` is not passed.

Reply:
If they are easy and not too time consuming right now, some other test cases for malformed config might make sense, but, at least we want the basic one of what happens if the flag is missing :)

Reply:
fe8992b6541de8c7128d8f621b399980c7a794e5

Added two guards, first the config path must be sent via error prone flag, and the config.xml file should contain a output dir value.